pkg/image/api/helper.go

```diff
@@ -50,6 +51,49 @@ func parseRepositoryTag(repos string) (string, string, string) {
 	return repos, "", ""
 }
 
+// ParseImageStreamImageName splits a string into its name component and ID component, and returns an error
+// if the string is not in the right form.
+func ParseImageStreamImageName(input string) (name string, id string, err error) {
+	segments := strings.Split(input, "@")
+	switch len(segments) {
+	case 2:
+		name = segments[0]
+		id = segments[1]
+		if len(name) == 0 || len(id) == 0 {
+			err = fmt.Errorf("neither name nor id part of image stream image name may be empty")
```

Comment:
This message doesn't make sense.  `image stream image names must have a name and ID`